### DISCUSSION TITLE
Maximize without gap?

### DISCUSSION THREAD

**Discussion by avbk** [rxhanson/Rectangle#440] [Ideas]
Hi, first of all thanks for the awesome work, Rectangle really works like a charm!

I'm using Rectangle on a daily basis and I set the window gap to 8px, because it pleases my eyes to see the windows having a small padding. But whenever I toggle to maximized it seems unnecessary to have a gap. So I would suggest that the gap is not applied when maximizing? 

The easiest solution would be to set `WindowAction.maximize.gapsApplicable` to `.none` (currently it is set to `.both` see [WindowAction.swift#L379](https://github.com/rxhanson/Rectangle/blob/master/Rectangle/WindowAction.swift#L379)).
An even nicer, but more complex, solution would be to add a boolean Setting and let the user decide whether he wants gaps on maximized windows or not?

I'm new to macos, but I could create a PR for the easy solution I guess, if you're interessted?

Best regards
Alex

**Comment by rxhanson** [comment]
Sure, I look forward to reviewing your PR. Thanks for looking to contribute. It can be another `BoolDefault` in `Defaults` and therefore toggleable via terminal command.

**Reply by avbk** [reply]
Cool, thanks for the hint. I hope to find some time on the weekend 🙂
